web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -29,6 +29,33 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
     daily.date.toUTC().toMillis()
   )
 
+  const calculateDailyPrepLevels = () => {
+    const prepLevels: (number | undefined)[] = []
+    range(NUM_WEEK_DAYS).map(day => {
+      const meanIntensityGroup = dailyMeanIntensityGroups[day]
+      prepLevels.push(calculatePrepLevel(meanIntensityGroup))
+    })
+    return prepLevels
+  }
+  const calculateMeanPrepLevel = (
```

Comment:
Can we move these functions outside of the component and test them? It's hard to see what values are closed over and which are passed in. They can go in `web/src/features/hfiCalculator/components/prepLevel.ts` for now.

Reply:
Agreed - HFI calculations shouldn't be happening in a cell.